include/networkit/io/NetworkitBinaryWriter.hpp

```diff
@@ -30,13 +30,20 @@ class NetworkitBinaryWriter final : public GraphWriter {
 
 public:
     NetworkitBinaryWriter(uint64_t chunks = 32,
-                          NetworkitBinaryWeights weightsType = NetworkitBinaryWeights::autoDetect);
+                          NetworkitBinaryWeights weightsType = NetworkitBinaryWeights::autoDetect,
+                          bool ignoreEdgeIndex = false);
```

Comment:
I don't like this `bool` flag here. Do we need it?

Reply:
Not really I could remove it and make always and only save indices of previously indexed graphs.